--- my.cnf.bkp	2015-11-18 20:45:16.000000000 -0800
+++ my.cnf	2015-11-18 20:53:11.000000000 -0800
@@ -29,6 +29,8 @@
 nice		= 0
 
 [mysqld]
+skip-host-cache
+skip-name-resolve
 user		= mysql
 pid-file	= /var/run/mysqld/mysqld.pid
 socket		= /var/run/mysqld/mysqld.sock
@@ -41,9 +43,9 @@
 
 # Instead of skip-networking the default is now to listen only on
 # localhost which is more compatible and is not less secure.
-bind-address	= 127.0.0.1
+# bind-address	= 127.0.0.1
 
-log-error	= /var/log/mysql/error.log
+# log-error	= /var/log/mysql/error.log
 
 # Disabling symbolic-links is recommended to prevent assorted security risks
 symbolic-links=0
